flake.nix

```diff
@@ -8,7 +8,7 @@
 
   inputs = {
     nixpkgs.url = "nixpkgs/nixos-unstable";
-    nixpkgsV1.url = "nixpkgs/nixos-22.11";
+    nixpkgsV1.url = "nixpkgs/nixos-unstable";
```

Comment:
Input change might be worth cherry-picking into a separate PR for visibility in history.

Reply:
done, see https://github.com/nix-community/dream2nix/pull/532